io/post_regional.F90

```diff
@@ -1940,7 +1953,7 @@ subroutine set_postvars_regional(wrt_int_state,mpicomp,setvar_atmfile,   &
                 do i=ista, iend
                   if (arrayr42d(i,j) < spval) then
                     isltyp(i,j) = nint(arrayr42d(i,j))
-                    if( abs(arrayr42d(i,j)-fillValue) < small)  isltyp(i,j) = spval
+                    if( abs(arrayr42d(i,j)-fillValue) < small)  isltyp(i,j) = 0
```

Comment:
I assume 0 is not valid soil type, i.e. it's a 'special' value. Is that correct?

Reply:
Right, the code follows standalone post to use zero for undefined value.